relays/bin-substrate/src/messages_target.rs

```diff
@@ -166,10 +179,13 @@ where
 		id: TargetHeaderIdOf<P>,
 	) -> Result<(TargetHeaderIdOf<P>, P::MessagesReceivingProof), SubstrateError> {
 		let (id, relayers_state) = self.unrewarded_relayers_state(id).await?;
+		let inbound_data_key = pallet_bridge_messages::storage_keys::inbound_lane_data_key::<R, I>(&self.lane_id);
```

Comment:
This will cause issues for Rococo & Westend. Can we avoid depending on the runtime directly and just add a function to `bp_rialto/bp_millau/bp_messages` crates which generates the storage key?

Reply:
I guess it's good for now, there is probably a bunch of other things to fix first anyway.

Reply:
I really wanted to avoid manually crafting storage keys. And the main thing is that we don't need runtime here at all - it's here just to satisfy compiler - only instance is used to build key. I'll see what can be done here

Reply:
So as [this comment](https://github.com/paritytech/parity-bridges-common/blob/7dace6246ec438ea025ad042270f049c1c2ccbe1/modules/messages/src/lib.rs#L656) explains, we may use any runtime that implements `Config<I>`. Presumably for all W/R/P/K chains we'll be using the same instance => we may use e.g. `millau_runtime::Runtime` everywhere. Alternatively I may just copypaste [this code](https://github.com/paritytech/substrate/blob/dcce8b4cfb461cbb2033021926e623b52c95fc74/frame/support/src/storage/generator/map.rs#L74) to the messages pallet, but as I said I wanted to avoid that.